zkevm-circuits/src/evm_circuit/execution/create.rs

```diff
@@ -310,8 +310,7 @@ impl<F: Field, const IS_CREATE2: bool, const S: ExecutionState> ExecutionGadget<
                     }
 
                     cb.require_step_state_transition(StepStateTransition {
-                        // +1: move to new context
```

Comment:
Just question asking : why original got `1.expr()` and remove here. I guess it's because the removal of `(CallContextField::Value, callee.value)`  ?

Reply:
Yes, it's bsc of the removal of `(CallContextField::Value, callee.value)`. I misunderstood the `+1` in the original pr.